### PR TITLE
python-packages: aiohttp-0.18.3, http client/server for asyncio

### PR DESCRIPTION
**Pull request by ldesgoui**

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -284,6 +284,31 @@ let
...
 
+
+  aiohttp = buildPythonPackage rec {
+    name = "aiohttp-${version}";
+    version = "0.18.3";
+    disabled = pythonOlder "3.3";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/a/aiohttp/${name}.tar.gz";
+      md5 = "9b0d7df0613a9dd68b40ddc1a36c4bec";
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: no need for `md5` (deprecated)

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -284,6 +284,31 @@ let
...
+      sha256 = "d3a0bb94538a496aaaa49ed647f36ab2d43f1867e018049badc941b7d6f34ade";
+    };
+
+    propagateBuildInputs = [ self.chardet ];
+
+    doCheck = false;
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: Could you write a comment why are there no tests? That's suprising.

Comment (ldesgoui) [inline pkgs/top-level/python-packages.nix]: Oh yeah, I had forgot about that. The tests have a dependency (pytest-raisesregexp) that's not in nixpkgs yet.

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: We use `sha256` nowadays

**Comment by domenkozar** [comment]
@ldesgoui this is almost ready :-)

**Comment by ldesgoui** [comment]
Great, thanks! I should have taken the time to get the tests working, but I'm too busy at the moment, sorry :(

**Comment by FRidh** [comment]
I hadn't noticed this PR. `aiohttp` and dependencies have been added in https://github.com/NixOS/nixpkgs/pull/11600
Guess we can close this.

**Comment by ldesgoui** [comment]
Thanks!

**Pull request closed without merging**